File: C++/Remove-Duplicates-from-Sorted-Array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int cur = -1;
        for (int i = 0; i < nums.size(); i ++) {
            if (cur == -1 || nums[i] != nums[cur]) {
                nums[++cur] = nums[i];
            }
        }
        return cur + 1;
    }
};

File: Python/Remove-Duplicates-from-Sorted-Array.py
class Solution:
    # @param a list of integers
    # @return an integer
    def removeDuplicates(self, A):
        length = len(A)
        if length <= 0:
            return 0
        num = 1;
        cnt = A[0]
        for i in range(1, length):
            if A[i] != cnt:
                cnt = A[i]
                A[num] = cnt
                num = num + 1
        return num